modules/atf_logger.lua

```diff
@@ -14,20 +32,28 @@ local Logger =
   hmi_log_format = '',
   start_file_timestamp = 0,
   mt = {
-    __index={}
+    __index = {}
   }
 }
 
 Logger.mobile_log_format = "%s(%s) [version: %s, frameType: %s, encryption: %s, serviceType: %s, frameInfo: %s, messageId: %s] : %s \n"
 Logger.hmi_log_format = "%s[%s] : %s \n"
 
-function Logger.formated_time(withoutDate)
-  if withoutDate == true then
+--- Create string representation of current time in set format
+-- @tparam ?boolean withoutDate Set date format
```

Comment:
@VVeremjova Yes it is used for optional parameters. 
In other words:
 `--@tparam ?boolean withoutDate` and `@tparam boolean|nil withoutDate` the same.